### ISSUE TITLE
Dialogs with Multifield and Sidebar saving conflict

### ISSUE THREAD

**Issue by wildone** [websight-io/starter#159]
**Description**
Saving a dialog with a multifield gets overwritten rogue save from sidebar wipes changes.

**Steps to reproduce**
1. Create a dialog with only a multifield
2. Add it to page
3. Update via dialog
4. Update via sidebar

**Current result**
Currently, when you save via dialog correct values are saved, but a rogue save wipes data.

**Expected behavior**
If dialog save occurs not additional saves should occur.

**Screenshots**
![dialogandsidebarsaveconflict](https://user-images.githubusercontent.com/170312/232957233-a64645ab-16a0-4bee-9f49-16a04fac1d54.gif)

**WebSight CMS version**
1.12.0

**Comment by paweljozwicki** [comment]
Thanks for noticing the issue.

**Comment by wildone** [comment]
Bit of a problem now, any official way to disable sidebar?

Can't deploy and edit content afterward as it disappears. 


![dialogandsidebarsaveconflict2](https://user-images.githubusercontent.com/170312/234473650-56f99b03-0c73-43f9-b407-b00baceb0946.gif)

This is just clicking on components without any updates, it's just wiping content.

![dialogandsidebarsaveconflict3](https://user-images.githubusercontent.com/170312/234474510-e9699562-1bfc-4d28-9142-bfaa9a03d1bd.gif)

Sidepanel should be doing PATCH requests only to update fields as they come out of focus. Right not it's just doing silly stuff.

**Comment by paweljozwicki** [comment]
Hi, the quickest workaround, for now, would be using `"ws:disallowedContext": ["edit:panel"]` property on multifield:  
For example:

```
    "elements": {
        "sling:resourceType": "wcm/dialogs/components/multifield",
        "name": "elements",
        "label": "Elements",
        "ws:disallowedContext": ["edit:panel"],
        "labelText": {
          "sling:resourceType": "wcm/dialogs/components/textfield",
          "name": "label",
          "label": "Label"
    }
```

The field would be not available in the side panel, but still it would be possible to edit it in the dialog.

**Comment by paweljozwicki** [comment]
Hi, the issue was resolved in WebSight 1.14.0. See [release notes](https://www.websight.io/docs/release-notes/websight-cms/#dialogs)
